src/shared/components/mutationTable/column/MutantCopiesColumnFormatter.tsx

```diff
@@ -0,0 +1,110 @@
+import * as React from 'react';
+import DefaultTooltip from 'shared/components/defaultTooltip/DefaultTooltip';
+import {Mutation, ClinicalData} from "shared/api/generated/CBioPortalAPI";
+import styles from "./mutationType.module.scss";
+import getCanonicalMutationType from "shared/lib/getCanonicalMutationType";
+
+interface IMutationTypeFormat {
+    label?: string;
+    longName?: string;
+    className: string;
+    mainType: string;
+    priority?: number;
+}
+
+/**
+ * @author Avery Wang
+ */
+export default class MutantCopiesColumnFormatter
+{
+    /* Determines the display value by using the impact field.
+     *
+     * @param data  column formatter data
+     * @returns {string}    mutation assessor text value
+     */
+    public static getDisplayValue(data:Mutation[], sampleIdToClinicalDataMap:{[sampleId:string]:ClinicalData[]}|undefined):string
```

Comment:
style : elsewhere we open blocks on the same line as function (or class) definitions